kernelci/scheduler.py

```diff
@@ -31,6 +31,10 @@ def __init__(self, configs, runtimes):
 
     def get_configs(self, event, channel='node'):
         """Get the scheduler configs matching a given event"""
+        # if event type is str, quit
```

Comment:
A minor fix.
```suggestion
        # if event type is not dict, quit
```

Reply:
Thanks, i forgot it was placeholder for more proper comment, updated now